e2e/resources/python-code/git-ignore-check.py

```diff
@@ -0,0 +1,25 @@
+! git clone https://github.com/all-of-us/workbench-snippets.git /tmp/workbench-snippets
+
+! touch /tmp/workbench-snippets/should_be_ignored.png
+! touch /tmp/workbench-snippets/should_be_ignored.csv
+! touch /tmp/workbench-snippets/should_be_ignored.tsv
+! touch /tmp/workbench-snippets/should_be_ignored.CSV
+! touch /tmp/workbench-snippets/should_be_ignored.TSV
+! touch /tmp/workbench-snippets/should_be_visible.r
+! touch /tmp/workbench-snippets/should_be_visible.R
+! touch /tmp/workbench-snippets/should_be_visible.py
+! touch /tmp/workbench-snippets/should_be_visible.wdl
+! touch /tmp/workbench-snippets/should_be_visible.sh
+! touch /tmp/workbench-snippets/should_be_visible.md
+! mkdir -p /tmp/workbench-snippets/some_diretory/
+! touch /tmp/workbench-snippets/some_diretory/should_be_ignored.CSV
+! touch /tmp/workbench-snippets/some_diretory/should_be_visible.sh
```

Comment:
further nit: could you test `.ipynb` and `.rst` and `LICENSE` files bc we're explicitly allowing them in the gitignore

Reply:
Good catch. This was copied from the terra-docker test case - rereading further, the test had a number of deficiencies. I've refactored it to be more precise and added better assertion failure messages